src/cleanupCommand/cleanupManager.ts

```diff
@@ -1,81 +1,178 @@
+    const notCleanedAndSuccess = await this.jobManager.getSuccessNotCleanedJobs(ingestionJobType);
+
     for (let i = 0; i < notCleanedAndSuccess.length; i += this.discreteBatchSize) {
       const currentBatch = notCleanedAndSuccess.slice(i, i + this.discreteBatchSize);
       const blackListFlitteredBatch = this.sourceBlackList.length > 0 ? this.filterBlackListSourcesTasks(currentBatch) : currentBatch;
-      await this.sourcesProvider.deleteDiscretes(blackListFlitteredBatch);
+      this.logger.info({
+        msg: `Will execute cleanup to ${blackListFlitteredBatch.length} success jobs of type: '${ingestionJobType}'`,
+        batch: `${i + 1}/${Math.floor(notCleanedAndSuccess.length / this.discreteBatchSize + 1)}`,
+        jobIds: blackListFlitteredBatch.map((job) => job.id),
+      });
+      const sourcesToDelete = currentBatch.filter((discrete) => !this.sourceBlackList.includes(discrete.parameters.originDirectory));
+      const ignoredSources = currentBatch.filter((discrete) => this.sourceBlackList.includes(discrete.parameters.originDirectory));
+      this.logger.info({
+        msg: `Will execute sources deletion after excluded from blacklist'`,
+        sourcesToDelete: sourcesToDelete.length ? sourcesToDelete.map((source) => source.parameters.originDirectory) : [],
+        ignoredSources: ignoredSources.length ? ignoredSources.map((source) => source.parameters.originDirectory) : [],
+      });
+      const sourcesDirectories = this.getSourcesLocation(blackListFlitteredBatch);
+      await this.sourcesProvider.deleteDiscretes(sourcesDirectories);
       await this.jobManager.markAsCompletedAndRemoveFiles(currentBatch);
+      this.logger.info({
+        msg: `Complete and mark jobs as 'Completed' with file directories remove`,
+        batch: `${i + 1}/${Math.floor(notCleanedAndSuccess.length / this.discreteBatchSize + 1)}`,
+        jobIds: currentBatch.map((job) => job.id),
+      });
+    }
+  }
+
+  public async cleanSuccessfulSwappedLayersTasks(swappJobType: string): Promise<void> {
+    this.logger.info({ msg: `Running Cleanup for success ingestion swap update jobs of type: ${swappJobType}` });
+    const notCleanedAndSuccess = await this.jobManager.getSuccessNotCleanedJobs(swappJobType);
+    for (let i = 0; i < notCleanedAndSuccess.length; i += this.discreteBatchSize) {
+      const currentBatch = notCleanedAndSuccess.slice(i, i + this.discreteBatchSize);
+
+      // cleaning tiles of all success jobs excluding layer that been exporting on current iteration.
+      const notRunningExportFilteredBatch = await this.filterFromRunningExportJobs(currentBatch);
+      this.logger.info({
+        msg: `Will execute cleanup to ${notRunningExportFilteredBatch.length} success jobs of type: '${swappJobType}'`,
+        batch: `${i + 1}/${Math.floor(notCleanedAndSuccess.length / this.discreteBatchSize + 1)}`,
+      });
+      this.logger.debug({ jobIds: notRunningExportFilteredBatch.map((job) => job.id) });
```

Comment:
please unite the debug log into the info log

Reply:
done